src/redisearch_rs/inverted_index/src/index_result/raw.rs

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+use std::{ffi::c_char, fmt::Debug, marker::PhantomData};
+
+use enumflags2::{BitFlags, bitflags};
+use ffi::{RSDocumentMetadata, RSQueryTerm, RSYieldableMetric, t_docId, t_fieldMask};
+use low_memory_thin_vec::LowMemoryThinVec;
+
```

Comment:
It's unclear which part of the API is considered as raw, probably worth a commit.

Reply:
Done in https://github.com/RediSearch/RediSearch/pull/6643/commits/d4e760f2247883f016d0865f73035c58af8fd8cf